ansible/playbooks/aix.yml

```diff
@@ -14,9 +14,23 @@
 
   tasks:
     - block:
+      # Set standard variables
+        - name: Load AdoptOpenJDKs variable file
+          include_vars: adoptopenjdk_variables.yml
+      # Install bash if it's not already there
+        - name: Check if bash is installed
+          shell: bash --version >/dev/null 2>&1
+          register: does_bash_exist
+          ignore_errors: yes
+        - name: install bash
+          yum: name=bash state=present update_cache=yes
```

Comment:
Can bash not be added to the yum installer section?  Or is it needed for the filesystem configuration?

Reply:
@jdekonin - `AIX_filesystem_config.sh` has `#!/bin/bash` at the top, so this would be a prereq for that (although that's an existing piece of code). I'd be tempted to re-engineer it in `sh`/`ksh` if that really is the only reason for having this extra section to install `bash`

Reply:
There should be filesystem configuration run every time this playbook is run, its a one time setup action, much like the openssl and initial yum install.  I wonder if these steps/actions would be better broken out and added as a basic setup document before the playbook is thrown at it.

Either way, I guess this is a discussion for a improvement beyond this commit and can be taken to a new issue, same with my suggestion to cpan below.

Reply:
Yep let's separate that out into a separate issue.

Reply:
Done: https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/655